.travis.yml

```diff
@@ -48,7 +48,7 @@ script:
   # We are using && below on purpose
   # ciPerformRelease must run only when the entire build has completed
   # This guarantees that no release steps are executed when the build or tests fail
-  - ./gradlew build idea -s -PcheckJavaCompatibility && ./gradlew ciPerformRelease
+  - ./gradlew build idea -s && ./gradlew ciPerformRelease
```

Comment:
Probably in the long run we want to check the compatibility with Java 8

Reply:
We can revisit that later, if we can no longer build with Java 8 on Travis. For now, we are building on JDK 8, so we should be compatible by default.